src/scripts/main.js

```diff
@@ -1,3 +1,20 @@
 'use strict';
 
-// write your code here
+function copyTabCol() {
+  const table = document.querySelector('table');
+  const rows = table.querySelectorAll('tr');
```

Comment:
Similarly, ensure that `rows` is not empty before proceeding with the loop. If there are no rows, the subsequent operations will not be necessary.